README.md

```diff
@@ -37,33 +50,21 @@ $ kubectl get ovms
 $ kubectl get ovms -o yaml testvm
 
 # To start an offline VM you can use
-$ kubectl patch offlinevirtualmachine testvm --type merge -p '{"spec":{"running":true}}'
+$ ./virtctl console --kubeconfig ~/.kube/config start testvm
```

Comment:
Is `--kubeconfig` still required?

@rmohr had some cobra patches which should solve this

Reply:
No, not required anymore. It will pick up `~/.kube/config`, if not overwritten with one of the other methods which are also available for `kubectl`.

Reply:
Requires virtctl 0.4.1 or newer.

Reply:
Ack - that's also set as the version to be used.

Reply:
addressed